src/main/java/com/derongan/minecraft/deeperworld/listeners/MovementListener.kt

```diff
@@ -90,6 +92,30 @@ object MovementListener : Listener {
                 if (!toSection.region.contains(correspondingPos.blockX, correspondingPos.blockZ)
                     || !inSpectator && correspondingPos.block.type.isSolid
                 ) {
+                    if(current.isOnTopOf(toSection)){
+                        from.block.type = Material.BEDROCK
+                        val spawnedBedrock = from.block
+
+                        // Keep bedrock spawned if there are players within a 1.5 radius (regular jump height).
+                        // If no players are in this radius, destroy the bedrock.
+                        deeperWorld.schedule{
+                            this.repeating(1)
+                            while(spawnedBedrock.location.up(1).getNearbyPlayers(1.5).isNotEmpty()){
```

Comment:
Would it be possible to fix issues on restart or reload by checking if the plugin is enabled, then skipping the yield if it's read as disabled.

Reply:
On restart/reload the scheduled tasks are all cleared anyway right? This task won't always be running on startup, there is one task for each temporary bedrock. Right now that means that if you were to be standing on temporary bedrock when the server restarts or plugin reloads, the bedrock will stay there since it has no task checking for it to be cleared anymore. I'll make it so that all temporary bedrock is cleared on plugin disable.

Reply:
eh honestly not that big of a deal, i'll accept